specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-05-01-privatepreview/examples/AlertProcessingRules_Create_or_update_add_correlation.json

```diff
@@ -0,0 +1,114 @@
+{
+  "parameters": {
+    "api-version": "2023-05-01-privatepreview",
+    "subscriptionId": "subId1",
+    "resourceGroupName": "alertscorrelationrg",
+    "alertProcessingRuleName": "CorrelationExample",
+    "alertProcessingRule": {
+      "location": "Global",
+      "tags": {},
+      "properties": {
+        "scopes": [
+          "/subscriptions/subId1"
+        ],
+        "actions": [
+          {
+            "actionType": "Correlation",
```

Comment:
I think we should go for something more descriptive, like the other actions. Maybe CorrelateAlerts?